assets/css/homepage.css

```diff
@@ -0,0 +1,530 @@
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 22px;
+  letter-spacing: 0;
+}
+
+.program-link-1 {
+  text-decoration: underline;
+}
+
+.program-link-2 {
+  display: none;
+  background: #ec5242;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  margin-top: 7px;
+}
+
+@media screen and (max-width: 768px) {
+  #program {
+    padding: 10px 20px;
+    justify-content: flex-start;
+  }
+
+  .program-link-1 {
+    display: none;
+  }
+
+  .program-link-2 {
+    display: flex;
+  }
+
+  .program-header {
+    margin-bottom: 30px;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .program-info {
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .program-card {
+    flex-direction: row;
+    padding: 5px;
+    margin-bottom: 5px;
+    margin-right: 0;
+    height: auto;
+    background-color: rgba(213, 216, 222, 0.1);
+  }
+
+  .program-image {
+    width: 55px;
+    justify-content: flex-start;
+  }
+
+  .program-image img {
+    width: 30px;
+    height: 30px;
+    margin-right: 5px;
+  }
+
+  .program-title h4 {
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 20px;
+    width: 80px;
+    text-align: left;
+  }
+
+  .program-desc p {
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 15px;
+    text-align: left;
+  }
+
+  .program-image,
+  .program-desc,
+  .program-title {
+    margin-bottom: 0;
+  }
+}
+
+#speakers {
+  padding: 40px 140px;
+  background: #d3d3d3;
+}
+
+.speakers-header {
+  margin-bottom: 30px;
+  display: flex;
+  justify-content: center;
+}
+
+.speakers-header h2 {
+  color: #272a31;
+}
+
+.speakers-header h2::after {
+  margin-left: 75px;
+}
+
+.speakers {
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-gap: 30px;
+  width: 100%;
+}
+
+.speaker {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+.speaker-img {
+  background: url('../img/picsbg.JPG') no-repeat top left;
+  background-size: 50% 50%;
+  padding-left: 12px;
+  padding-top: 12px;
+  margin-right: 15px;
+}
+
+.speaker-img img {
+  width: 120px;
+  height: auto;
+}
+
+.speaker-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+.speaker-name h5 {
+  color: #272a31;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 25px;
+  letter-spacing: 0;
+  margin-top: 12px;
+}
+
+.speaker-title h6 {
+  color: #ec5242;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 400;
+  font-size: 21px;
+  line-height: 22px;
+  letter-spacing: 0;
+}
+
+.speaker-name,
+.speaker-title {
+  margin-bottom: 15px;
+}
+
+.speaker-message p {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly, please redesign your app using a mobile-first approach, as learned in the whole module that we finish now.